facefusion/processors/frame/modules/face_swapper.py

```diff
@@ -215,8 +219,15 @@ def swap_face(source_face : Face, target_face : Face, temp_frame : Frame) -> Fra
 				frame_processor_inputs[frame_processor_input.name] = prepare_source_embedding(source_face)
 		if frame_processor_input.name == 'target':
 			frame_processor_inputs[frame_processor_input.name] = crop_frame
-	crop_frame = frame_processor.run(None, frame_processor_inputs)[0][0]
-	crop_frame = normalize_crop_frame(crop_frame)
+	crop_frame = frame_processor.run(None, frame_processor_inputs)[0]
+	iterations = int(numpy.ceil(frame_processors_globals.face_swapper_weight))
+	blend = frame_processors_globals.face_swapper_weight % 1.0
```

Comment:
blend is not a good name -

Reply:
what about `remaining_factor`? which is what it is